README.md

```diff
@@ -1 +1,12 @@
-# eos-icons-vsc
\ No newline at end of file
+# EOS Icons for VSCode
+
+This extension allows you to replace VSCode's default icons with those from the
+[EOS icon set](https://eos-icons.com/).
+
+
+## Usage
+
+Install the extension from the Marketplace, execute the command `Preferences:
```

Comment:
is this testable at this stage or is this just a placeholder for when the extension (theme) is published?

Reply:
The command execution can be tested, but the actual result is not easily testable, as it would require some kind of test runner to do image recognition.

Reply:
I was wrong, it might be actually testable that the correct icon is used. But I would prefer to do that in a later PR, because that will be a lot more work.